### PR TITLE
add configuration for block and allow-list pf support entries in the warp menu

### LINKED ISSUE #53: Add block and allow-list fur support entries in the warp menu

### PR DESCRIPTION
**Pull request by kahoona77**
resolves #53

### CODE REVIEW

File: controllers/warp/configReader.go [hunk 1]
```diff
@@ -104,35 +119,48 @@ func (reader *ConfigReader) dogusReader(source config.Source) (types.Categories,
 	return reader.createCategories(dogus), nil
 }
 
-func (reader *ConfigReader) getDisabledSupportIdentifiers() ([]string, error) {
-	disabledSupportEntries, err := reader.registry.Get(disableWarpSupportEntriesConfigurationKey)
+func (reader *ConfigReader) readStrings(registryKey string) ([]string, error) {
+	entry, err := reader.registry.Get(registryKey)
```
Comment (cesmarvin) [inline controllers/warp/configReader.go:123]: 🚫 **[checkstyle]** <typecheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>reader.registry.Get undefined (type watchConfigurationContext has no field or method Get)

File: controllers/warp/configReader.go [hunk 2]
```diff
@@ -104,35 +119,48 @@ func (reader *ConfigReader) dogusReader(source config.Source) (types.Categories,
...
+	err = json.Unmarshal([]byte(entry), &strings)
+	if err != nil {
+		return []string{}, fmt.Errorf("failed to unmarshal etcd key to string slice: %w", err)
+	}
+
+	return strings, nil
+}
+
+func (reader *ConfigReader) readBool(registryKey string) (bool, error) {
+	entry, err := reader.registry.Get(registryKey)
```
Comment (cesmarvin) [inline controllers/warp/configReader.go:138]: 🚫 **[checkstyle]** <typecheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>reader.registry.Get undefined (type watchConfigurationContext has no field or method Get)

Comment (kahoona77) [reply controllers/warp/configReader.go:138]: That is not true => False-Positive

**Review by sklein94 — APPROVED** [review]
LGTM

**Review by sklein94 — APPROVED** [review]
LGTM

**Pull request merged**